tests/containers/firewall.pm

```diff
@@ -60,6 +60,11 @@ sub run {
 sub post_fail_hook {
     my ($self) = @_;
 
+    if ($runtime eq "docker") {
+        systemctl('status docker.service');
```

Comment:
systemctl status docker.service gives EXITCODE 3 when the service is not active - which in turn means, again, cancel the post fail hook here.